src/TTextEdit.cpp

```diff
@@ -249,6 +251,7 @@ void TTextEdit::updateScreenView()
     // This was "if (pC->mType == TConsole::MainConsole) {"
     // and mIsMiniConsole is true for user created Mini Consoles and User Windows
     if (mpConsole->getType() == TConsole::MainConsole) {
+        mFontMetrics = QFontMetrics(mpHost->mDisplayFont);
         mFontWidth = QFontMetrics(mpHost->mDisplayFont).width(QChar('W'));
```

Comment:
Someone else (possibly me) also has a PR pending that will change this to:
```
    mFontWidth = QFontMetrics(mpHost->mDisplayFont).averageCharWidth();
```
so there may be git conflicts to resolve in the near future...

Reply:
And of course this and the following lines can all reuse the newly found `mFontMetrics` class instance and refer to that instead of each of them fetching a new copy...

Reply:
My idea is to completely remove `mFontWidth`: it's an approssimation of the char width but we could use the exact width instead. I just can't completely understand some usages of that variable and thus kept the original code.
The solution I'm proposing in this branch isn't perfect yet, but I'll work on it if it could be accepted.

Reply:
I agree, `mFontWidth` does not make too much sense anymore, however I'm not sure about converting everything to work with pixels either just yet - because that breaks a lot of assumptions we've made so far... :sweat_smile: this will require some thinking!